db/install.xml

```diff
@@ -29,6 +29,7 @@
         <FIELD NAME="description" TYPE="text" NOTNULL="false" SEQUENCE="false" COMMENT="Description of the custom content type."/>
         <FIELD NAME="descriptionformat" TYPE="int" LENGTH="4" NOTNULL="false" DEFAULT="0" SEQUENCE="false" COMMENT="The format of the description field of the custom content type."/>
         <FIELD NAME="datasources" TYPE="text" NOTNULL="false" SEQUENCE="false" COMMENT="Datasources used by this cms type, comma separated."/>
+        <FIELD NAME="instance_name" TYPE="char" LENGTH="255" NOTNULL="true" SEQUENCE="false" COMMENT="The name of this custom content type instance."/>
```

Comment:
Shouldn't we name this something like `name_template` or something to that effect? Since what we are actually storing is the mustache template that makes up the name field right? Not the actual name string itself derived from the mustache template?

Reply:
Personally I think 'mustache' should be changed to 'instance_content', but whatever. As for text, it is 'char' to be consistent with name fields in other tables.

Reply:
Yeah I suppose, but seeing as what the field is, is basically storing the mustache template for the title/name, I'd be more inclined to have the database field match. Same with the char thing, what if we had a mustache template that was somewhat complex and exceeded the 255 chars (not the name/title itself mind you but the definition)

Reply:
done